### ISSUE TITLE
got Assertion failed: (!ctx.ssavalue_assigned.at(idx)), function emit_ssaval_assign by an empty keyword arguments

### ISSUE THREAD

**Issue by wookay** [JuliaLang/julia#36774]
```julia
julia> function f(; kwargs...)
           a = (kwargs...,)
           @info :msg
       end
f (generic function with 1 method)

julia> f()
Assertion failed: (!ctx.ssavalue_assigned.at(idx)), function emit_ssaval_assign, file /Users/wookyoung/head/julia/src/codegen.cpp, line 3868.

signal (6): Abort trap: 6
in expression starting at REPL[2]:1
__pthread_kill at /usr/lib/system/libsystem_kernel.dylib (unknown line)
Allocations: 752077 (Pool: 751930; Big: 147); GC: 1
[1]    82682 abort      /Users/wookyoung/head/julia/usr/bin/julia
```

1. with keyword arguments works fine.
```
julia> f(k=1)
[ Info: msg
```

2. using `println` works fine instead of `@info`.
```
julia> function f(; kwargs...)
           a = (kwargs...,)
           println(:msg)
       end
f (generic function with 1 method)

julia> f()
msg
```

```
julia> versioninfo()
Julia Version 1.6.0-DEV.520
Commit 7eecb73f29* (2020-07-23 07:03 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin19.5.0)
  CPU: Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, skylake)
Environment:
  JULIA_CUDA_SILENT = 1
```

**Comment by JeffBezanson** [comment]
Works in 1.5.

**Comment by wookay** [comment]
fixed by #37634
